ui/admin/app/routes/scopes/scope/sessions/index.js

```diff
@@ -81,10 +87,17 @@ export default class ScopesScopeSessionsIndexRoute extends Route {
     if (!this.allSessions) {
       await this.getAllSessions(scope_id);
     }
-    if (!this.associatedUsers) {
+    if (
+      this.can.can('list model', globalScope, { collection: 'users' }) &&
+      this.can.can('list model', orgScope, { collection: 'users' }) &&
```

Comment:
Can these permission checks be moved into the helper methods since they're used multiple times?

Reply:
Yep.